_posts/2020-09-21-porting-namecoin-in-tor-browser-to-macos.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Porting Namecoin in Tor Browser to macOS"
+author: Jeremy Rand
+tags: [News]
+---
+
+Hot on the heels of [porting Namecoin in Tor Browser to Windows]({{ "/2020/08/28/porting-namecoin-in-tor-browser-to-windows.html" | relative_url }}), it was time to do the final remaining desktop OS, macOS.  This one was always going to present a challenge in the QA testing department, because I don't own any macOS devices.  On the other hand, I fully expected that the amount of code written for macOS support was going to be pretty minimal; in fact I expected that other than tweaking a few constants, macOS support was likely to be a subset of the code needed for Windows support.  But of course, our friend Murphy wasn't going to let that assumption stand.
```

Comment:
>our friend Murphy wasn't going to let that assumption stand.

our friend Mr. Murphy wasn't going to let that assumption stand.

Reply:
I dislike titles, so I'm not going to add `Mr.` here.

Reply:
I do think it should be indicated that it's a family name and not a given name, but since I don't see any way to do that without honorifics, I won't insist on it.